.openpublishing.redirection.json

```diff
@@ -406,9 +406,13 @@
             "source_path": "docs/csharp/language-reference/proposals/csharp-7.0/index.md",
             "redirect_url": "/dotnet/csharp/language-reference/proposals/csharp-7.0/pattern-matching"
         },
+        {
+            "source_path": "docs/csharp/language-reference/proposals/csharp-7.1/index.md",
+            "redirect_url": "/dotnet/csharp/language-reference/proposals/csharp-7.1/async-main"
+        },
         {
             "source_path": "docs/csharp/language-reference/proposals/index.md",
-            "redirect_url": "/dotnet/csharp/language-reference/proposals/csharp-7.0/pattern-matching"
+            "redirect_url": "/dotnet/csharp/language-reference/proposals/csharp-7.1/async-main"
```

Comment:
shouldn't this still go to the old URL or do you want to go to the latest?

Reply:
That should go to the latest (with each update)

Reply:
ok. so would it make more sense to list them from latest to older?